unconference/_toc.yml

```
@@ -5,6 +5,8 @@ format: jb-book
 root: intro
 chapters:
   - file: mentoring_guide_projects.md
+  - file: mentoring_onboarding_checklist.md
+  - file: mentoring_meeting_template.md
```

```suggestion
  - file: mentoring_guide_projects.md
    sections:
    - file: mentoring_onboarding_checklist.md
    - file: mentoring_meeting_template.md
```

I think they'd fit better as sections instead of chapters. That is, they won't appear directly on the left sidebar, instead the mentoring section will have an arrow to "unfold" itself and reveal its sections. The unfolding will also happen automatically whenever one is reading either of the 3 documents. Reference and example: https://jupyterbook.org/en/stable/structure/toc.html#use-chapter-sub-sections

---

Good call @OriolAbril, I wanted it to be sessions and didn't know the way, let's merge your suggestions